scripts/bootstrap_docker.sh

```diff
@@ -51,7 +51,7 @@ if [[ "${flannelCID}" == "" ]]; then
               --privileged \
               -v /dev/net:/dev/net \
               --name=flannel \
-              quay.io/coreos/flannel:latest /opt/bin/flanneld -iface="${interface}" -etcd-endpoints="${connection_string}")
+              quay.io/coreos/flannel:0.5.5 /opt/bin/flanneld -iface="${interface}" -etcd-endpoints="${connection_string}")
```

Comment:
LGTM  - ultimately we'll need to surface this as a config option so the end user can update the flannel deployment... but that seems like a larger change than whats proposed here which fixes an immediate concern. 👍